addons/survey/static/src/js/survey.js

```diff
@@ -169,11 +168,10 @@ if(!the_form.length) {
         return ajax.loadJS(url);
     }
 
-    var ready_with_locale = $.when(base.ready(), load_locale());
     // datetimepicker use moment locale to display date format according to language
     // frontend does not load moment locale at all.
     // so wait until DOM ready with locale then init datetimepicker
-    ready_with_locale.then(function(){
+    load_locale().then(function(){
```

Comment:
Should still wait for dom ready here

Reply:
https://github.com/odoo/odoo/pull/30409/files/148345e5815f6d49ab04a8a0d404bcabc5e0a757#diff-55a36bd725045e96a61a869da9487453L4

Reply:
Oh. Lol.

Reply:
:laughing: